topics/clang.md

```diff
@@ -306,15 +308,15 @@ As mentioned in [prerequisites](#Prepare+your+project), `qodana-clang` reads the
                     - name: 'Qodana Scan'
                       uses: JetBrains/qodana-action@v2024.3
                       with:
-                        args: --linter,%qp-linter%,--compile-commands,&lt;path-to-compile_commands.json&gt;
+                        args: --linter,%qdcppc-image%,--compile-commands,&lt;path-to-compile_commands.json&gt;
                       env:
                         QODANA_TOKEN: ${{ secrets.QODANA_TOKEN }}
           </code-block>
 
   <p>More configuration examples are available in the <a href="github.md"/> section.</p>
     </tab>
     <tab title="Jenkins" group-key="jenkins">
-        <note>This feature is in experimental mode, which means that its operation can be unstable.</note>
+        <note>This feature is experimental, which means that is actively in development and should not be used in a production environment.</note>
```

Comment:
```suggestion
        <note>This feature is experimental, which means it is actively in development and should not be used in a production environment.</note>
```

Reply:
Fixed

Reply:
I think that this should be rewritten the following way:

This feature is experimental and being actively developed, which means that it should not be used in a production environment.